pkgs/development/python-modules/catboost/default.nix

```diff
@@ -5,15 +5,15 @@
 
 buildPythonPackage rec {
   pname = "catboost";
-  version = "1.1.1";
+  version = "1.0.5";
```

Comment:
Please add a comment to prevent unintended updates of this package.
Also, add `# nixpkgs-update: no auto update` to prevent `@r-ryantm` to update this package

Reply:
Done, thanks.

Reply:
This was not done by r-ryantm, IIRC for python packages there is a list somewhere. Also the comment should be above the line.